underfs/hdfs/pom.xml

```diff
@@ -114,19 +114,6 @@
                     </includes>
                   </artifactSet>
                   <relocations>
-                    <!-- Relocation for non-Alluxio service loading files  -->
```

Comment:
we could not rename hadoop classes (but they are still included in the ufs jar), will discuss with you in person.